src/main/java/com/github/fishio/MenuField.java

```diff
@@ -13,6 +13,8 @@
 public class MenuField extends PlayingField {
 	private GameThread gameThread;
 	
+	private static final int MAX_ENEMIES = 100;
+	
```

Comment:
Actually, the size of these fish are a lot smaller than the previous version of fish-spawns in the main menu we had.
So based on that, it's fine. However, I do prefer the older amount and sizes of fish in the main menu, mainly because this version takes about 2 seconds to get running. Fish first spawn and only after a certain moment they actually start moving. It all looks a bit buggy, probably because spawning 100 fish and making them move at once is too much to handle for my computer.

So I'd rather have the old amount and sizes of fishes (which is like 10 fish max, but more size differentiation?).

<logic,performance,user_interface